Sun Sep 08 18:20:59 CAT 2019 : Apache Derby Network Server - 10.13.1.1 - (Unversioned directory) started and ready to accept connections on port 1527
----------------------------------------------------------------
Sun Sep 08 18:21:14 CAT 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.13.1.1 - (Unversioned directory): instance a816c00e-016d-11ad-b3c0-00007715c241 
on database directory C:\data\TUT\2019\Task\Databases\DoEDB with class loader sun.misc.Launcher$AppClassLoader@266474c2 
Loaded from file:/C:/data/glassfish-5.0/glassfish5/javadb/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_152-b16
user.dir=C:\data\glassfish-5.0\glassfish5\javadb
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\data\TUT\2019\Task\Databases
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Sun Sep 08 18:21:22 CAT 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.13.1.1 - (Unversioned directory): instance 9ac8804c-016d-11ad-b3c0-00007715c241 
on database directory C:\data\TUT\2019\Task\Databases\TUTDB with class loader sun.misc.Launcher$AppClassLoader@266474c2 
Loaded from file:/C:/data/glassfish-5.0/glassfish5/javadb/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_152-b16
user.dir=C:\data\glassfish-5.0\glassfish5\javadb
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\data\TUT\2019\Task\Databases
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Sun Sep 08 18:21:27 CAT 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.13.1.1 - (Unversioned directory): instance 513b408a-016d-11ad-b3c0-00007715c241 
on database directory C:\data\TUT\2019\Task\Databases\UJDB with class loader sun.misc.Launcher$AppClassLoader@266474c2 
Loaded from file:/C:/data/glassfish-5.0/glassfish5/javadb/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_152-b16
user.dir=C:\data\glassfish-5.0\glassfish5\javadb
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\data\TUT\2019\Task\Databases
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Sun Sep 08 18:27:12 CAT 2019:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.13.1.1 - (Unversioned directory): instance 4b6f00c8-016d-11ad-b3c0-00007715c241 
on database directory C:\data\TUT\2019\Task\Databases\CRDB with class loader sun.misc.Launcher$AppClassLoader@266474c2 
Loaded from file:/C:/data/glassfish-5.0/glassfish5/javadb/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_152-b16
user.dir=C:\data\glassfish-5.0\glassfish5\javadb
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\data\TUT\2019\Task\Databases
Database Class Loader started - derby.database.classpath=''
Sun Sep 08 18:39:59 CAT 2019 Thread[DRDAConnThread_14,5,main] Cleanup action starting
java.sql.SQLException: Database 'sample' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
Caused by: ERROR XJ004: Database 'sample' not found.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 21 more
============= begin nested exception, level (1) ===========
ERROR XJ004: Database 'sample' not found.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
============= end nested exception, level (1) ===========
Cleanup action completed
Sun Sep 08 18:39:59 CAT 2019 Thread[DRDAConnThread_14,5,main] (DATABASE = sample), (DRDAID = {17}), Database 'sample' not found.
----------------------------------------------------------------
Mon Sep 09 09:42:55 CAT 2019: Shutting down Derby engine
----------------------------------------------------------------
Mon Sep 09 09:42:55 CAT 2019:
Shutting down instance 4b6f00c8-016d-11ad-b3c0-00007715c241 on database directory C:\data\TUT\2019\Task\Databases\CRDB with class loader sun.misc.Launcher$AppClassLoader@266474c2 
